README.md

```diff
@@ -34,11 +24,30 @@ based on the [Bulletproofs](https://doc.dalek.rs/bulletproofs/index.html) zero-k
 * [Spacesuit README](spacesuit/README.md)
 * [Cloak specification](spacesuit/spec.md)
 
+### [Musig](musig)
+
+A pure Rust implementation of the [MuSig signature aggregation scheme](https://eprint.iacr.org/2018/068) by Maxwell, Poelstra, Seurin and Wuille.
```

Comment:
Let's keep the capitalization consistent - so maybe call it "Musig" or just "Simple Schnorr Multi-Signatures", as in the paper title

Reply:
fair